app/src/main/java/org/mozilla/reference/browser/media/MediaSessionService.kt

```diff
@@ -0,0 +1,12 @@
+package org.mozilla.reference.browser.media
```

Comment:
nit: Missing license header!

But I'll land anyways to get this into a Nightly build.

Reply:
oh oops, I'll fix.  Thanks for landing this.